index.js

```diff
@@ -269,6 +269,14 @@ function fromArrayLike (array) {
   return buf
 }
 
+function fromArrayView (arrayView) {
+  if (isInstance(arrayView, Uint8Array)) {
+    var copy = Uint8Array.from(arrayView)
```

Comment:
To increase compatibility with browsers that don't have `Uint8Array.from` I changed this line to:

`var copy = new Uint8Array(arrayView)`

I believe this shouldn't affect performance, but it would be great if you could confirm this, @koush.